lib/inertia_rails/rspec.rb

```diff
@@ -49,14 +52,17 @@ def inertia_wrap_render(render)
       protected 
       
       def inertia_tests_setup?
-        @_inertia_render_wrapper.present?
+        ::RSpec.current_example.metadata.fetch(:inertia, false)
       end
     end
   end
 end
 
 RSpec.configure do |config|
   config.include ::InertiaRails::RSpec::Helpers
+  config.add_setting :inertia, default: {
```

Comment:
created this as an inertia scoped hash in case it needs to grow in the future.

My quick research led me to `config.add_setting`. If there's a more RSpec-y way to do this, happy to use it.